apps/nextjs-website/src/lib/fetch/types/page.ts

```diff
@@ -0,0 +1,127 @@
+import * as t from 'io-ts';
+
+const HeroSectionCodec = t.strict({
+  __component: t.literal('sections.hero'),
+  title: t.string,
+  subtitle: t.union([t.string, t.null]),
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  inverse: t.boolean,
+  size: t.keyof({
+    small: null,
+    big: null,
+  }),
+  useHoverlay: t.boolean,
+  sectionID: t.union([t.string, t.null]),
```

Comment:
Is the `sectionID` used somehow?

Reply:
Short answer is yes.

Long answer is it will when sections will be implemented.
It's used for linking to a section inside a page like `example.com/cittadini#sectionID`